src/settings/subsonicsettingspage.cpp

```diff
@@ -90,6 +92,9 @@ void SubsonicSettingsPage::Load() {
       break;
   }
 
+  bool download_album_covers = ui_->checkbox_download_album_covers->isChecked();
```

Comment:
Juse use `ui_->checkbox_download_album_covers->isChecked()` directly instead of a variable here.

Reply:
Done